tests/CMakeLists.txt

```diff
@@ -1,15 +1,18 @@
 
 add_library(platform_test
-  ${CMAKE_CURRENT_SOURCE_DIR}/src/catch.cpp)
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/catch.cpp
+)
 
 target_link_libraries(platform_test
   PUBLIC
   platform_mock
-  -lpthread)
+  -lpthread
+)
 
 target_include_directories(platform_test
-    PUBLIC
-    ${CMAKE_CURRENT_SOURCE_DIR}/catch)
+  PUBLIC
+  ${CMAKE_CURRENT_SOURCE_DIR}/catch
+)
 
 file(GLOB TEST_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/unit/*.cpp)
```

Comment:
Probably should also not use `GLOB` here for the same reasons you mentioned.

Reply:
Yeah good point, I'll list out the test files here.